svgdigitizer/svgplot.py

```diff
@@ -208,17 +208,14 @@ def units(self):
         ...   </g>
         ... </svg>'''))
         >>> plot = SVGPlot(svg)
-        >>> with TestCase.assertLogs(_) as logs:
-        ...    plot.units
-        ...    print(logs.output)
+        >>> plot.units
         {'x': 'm', 'y': 'A'}
-        ['WARNING:svgplot:Units on y axis do not match. Will ignore unit mA and use A.']
```

Comment:
I am not sure what happened here. But I think that this was an important test that we do not want to loose.